### PR TITLE
Add spec_url for CSS selectors A-F

### PR DESCRIPTION
**Pull request by sideshowbarker**

### CODE REVIEW

File: css/selectors/active.json [hunk 1]
```diff
@@ -5,6 +5,10 @@
         "__compat": {
           "description": "<code>:active</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:active",
+          "spec_url": [
+            "https://html.spec.whatwg.org/multipage/scripting.html#selector-active",
+            "https://drafts.csswg.org/selectors/#active-pseudo"
```
Comment (foolip) [inline css/selectors/active.json:10]: https://drafts.csswg.org/selectors/#the-active-pseudo is the section, if we're still doing that.

Comment (sideshowbarker) [reply css/selectors/active.json:10]: > [drafts.csswg.org/selectors/#the-active-pseudo](https://drafts.csswg.org/selectors/#the-active-pseudo) is the section, if we're still doing that.

Yeah I guess we need to decide. But for all features in this patch, what’s currently here was just automatically scraped out from the Specification table in the MDN article for the feature.

Comment (foolip) [reply css/selectors/active.json:10]: @Elchi3 did we decide? :)

Comment (Elchi3) [reply css/selectors/active.json:10]: I think we decided for the section as it is better UX. I haven't heard any other argument, so I was fine with deciding on sections.

File: css/selectors/after.json [hunk 2]
```diff
@@ -5,6 +5,10 @@
         "__compat": {
           "description": "<code>::after</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::after",
+          "spec_url": [
+            "https://drafts.csswg.org/css-pseudo/#selectordef-after",
+            "https://drafts.csswg.org/selectors-3/#gen-content"
+          ],
```
Comment (Elchi3) [inline css/selectors/after.json:11]: Seems to me like the selectors level 3 definition is outdated.


```suggestion
          "spec_url": "https://drafts.csswg.org/css-pseudo/#generated-content",
```

File: css/selectors/before.json [hunk 3]
```diff
@@ -5,6 +5,10 @@
         "__compat": {
           "description": "<code>::before</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::before",
+          "spec_url": [
+            "https://drafts.csswg.org/css-pseudo/#selectordef-before",
+            "https://drafts.csswg.org/selectors-3/#gen-content"
+          ],
```
Comment (Elchi3) [inline css/selectors/before.json:11]:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-pseudo/#generated-content"
```

File: css/selectors/active.json (hunk 1 above)
Comment (Elchi3) [inline css/selectors/active.json:10]:
```suggestion
            "https://drafts.csswg.org/selectors/#the-active-pseudo"
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]

File: css/selectors/first-letter.json [hunk 4]
```diff
@@ -5,6 +5,10 @@
         "__compat": {
           "description": "<code>::first-letter</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::first-letter",
+          "spec_url": [
+            "https://drafts.csswg.org/css-pseudo/#first-letter-pseudo",
+            "https://drafts.csswg.org/selectors-3/#first-letter"
+          ],
```
Comment (Elchi3) [inline css/selectors/first-letter.json:11]:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-pseudo/#first-letter-pseudo",
```

File: css/selectors/first-line.json [hunk 5]
```diff
@@ -5,6 +5,10 @@
         "__compat": {
           "description": "<code>::first-line</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::first-line",
+          "spec_url": [
+            "https://drafts.csswg.org/css-pseudo/#first-line-pseudo",
+            "https://drafts.csswg.org/selectors-3/#first-line"
+          ],
```
Comment (Elchi3) [inline css/selectors/first-line.json:11]:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-pseudo/#first-line-pseudo",
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Sorry, apparently I missed two dated specs here.

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**